docs/reference/api.yml

```
@@ -352,6 +352,27 @@ components:
       name: categories_tags_en
       description: |
         The category of products you are searching for.
+    sort_by:
+      schema:
+        type: string
+        example: product_name
+        enum:
+          - product_name
+          - last_modified_t
+          - last_modified_t_complete_first
+          - scans_n
+          - unique_scans_n
+          - created_t
+          - completeness
+          - popularity_key
+          - popularity
```

As popularity is the same as popularity_key is code, we may only keep the latter:
```suggestion
          - popularity_key
```